groups.py

```diff
@@ -768,6 +790,25 @@ def api_group_subcategories(ctx, category):
     return getSubcategories(ctx, category)
 
 
+@api.make()
```

Comment:
This API call fits better in schema.py I think.

Reply:
I guess you meant get_schemas here which indeed is moved from groups.py to schem.py 👍